tests/integration/test_integration_snapshots.py

```diff
@@ -273,3 +275,13 @@ def test_trace_with_wrong_metrics_types_not_sent(metrics):
                 with tracer.trace("child") as child:
                     child._metrics = metrics
         log.exception.assert_called_once_with("error closing trace")
+
+
+@snapshot()
+def test_tracetagsprocessor_only_adds_new_tags():
+    tracer = Tracer()
+    with tracer.trace(name="web.request") as span:
```

Comment:
should we be doing a `span.context.sampling_priority = <something that isn't user keep>`?

Reply:
`set_tag("manual.drop")` should handle that for us, but I do like the explicit `span.context.sampling_priority = AUTO_KEEP` or something.

Reply:
ah right, completely forgot lol..